Documentation/docs-site/content/usage/_index.en.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Using Fission"
+chapter: true
+draft: false
+weight: 40
+---
+
+# Using Fission
+
+### Examples, code samples and How To's
```

Comment:
Where does this show up?  Also, suggest rephrasing to "Usage guides, tutorials and examples".

Reply:
So this is a "chapter" title and it shows up like this: 
![image](https://user-images.githubusercontent.com/1167948/36463057-58beda8e-16ed-11e8-867a-6540bb01ea73.png)